Jenkinsfile

```diff
@@ -0,0 +1,84 @@
+pipeline {
+  agent {
+    dockerfile {
+      additionalBuildArgs '--build-arg USER_ID=$(id -u) --build-arg GROUP_ID=$(id -g)'
+    }
+  }
+  options {
+    ansiColor('xterm')
+  }
+  stages {
+    stage("Init title") {
+      when { changeRequest() }
+      steps {
+        script {
+          currentBuild.displayName = "PR ${env.CHANGE_ID}: ${env.CHANGE_TITLE}"
+        }
+      }
+    }
+    stage('Dependencies') {
+      steps {
+        sh '''
+          make deps
+        '''
+      }
+    }
+    stage('Build KEVM-VM') {
+      steps {
+        sh '''
+          make build-kevm-node
+        '''
+      }
+    }
+    stage('Build Ganache with KEVM-VM') {
+      steps {
+        script {
+          try {
+            sh '''
+              make ganache
```

Comment:
We can use `|| true` at the end of this line to avoid the more verbose `try...catch...`